events/elections/2020/README.md

```diff
@@ -31,24 +31,39 @@ previous [governance meeting video] which led to this whole process.
 
 ## Eligibility
 
-Please refer to the [Steering Committee Election Charter] for:
+Please refer to the [Steering Committee Election Charter] for [Eligibility for candidacy]
 
-- [Eligibility for candidacy]
-- [Eligibility for voting] and [Eligible voters] list (not in charter)
+Eligibility for voting in 2020 is defined as:
+
+* People who had at least 50 contributions to the Kubernetes project over
+  the past year, according to a snapshot taken 2020-08-01 of the data driving the [devstats developer activity counts dashboard][devstats-dashboard], who are also [Org Members].
+  Contributions include GitHub events like creating issues, creating pr's,
+  reviewing PR's, commenting on issues, etc. For full details see
+  [the SQL query used by devstats for developer activity counts][devstats-sql].
+
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. We *explicitly* believe the above heuristic will be
+  inaccurate and not represent the entire community. Thus we provide the form
+  for those who have contributed to the project but may not meet the above
```

Comment:
From last year's retro,

> Exemption forms: Make it **clear** that contributions listed -must- be related to the upstream Kubernetes project.

Can we explicitly state that here? IIRC folks who contributed to the Kubernetes end user ecosystem also ended up requesting an exception.

Reply:
The place to do that is in the form, not on the readme, so adding text to the form.

Reply:
here is the language I added to the form:

Please only describe contributions to upstream Kubernetes project(s) (including KIND and other infrastructure projects), not contributions to ecosystem projects and products.  As much as we love our ecosystem, those don't affect voting eligibility.